hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -399,9 +447,57 @@ protected void writeIncomingRecords() throws IOException {
     }
   }
 
+  protected GenericData.Record cdcRecord(CDCOperationEnum operation, String recordKey, String partitionPath,
```

Comment:
RFC-46 is moving away from GenericRecord as the canonical data record. So we may want to move in that direction as well. We need to sequence the two efforts correctly.